packages/node/src/lib/metrics-log.ts

```diff
@@ -132,13 +132,13 @@ export function metricsAPICall(
     makeRequest();
     return Promise.resolve({
       ids: getLogIds(body),
-    });
+    } as LogResponse);
```

Comment:
`Promise.resolve` has generics so you should be able to do this:

```
return Promise.resolve<LogResponse>({
  ids: getLogIds(body),
});
```

Reply:
nice, thanks! done here: https://github.com/readmeio/metrics-sdks/pull/900/commits/a718dae82a3577661b61fe6e7d7572d1715bfabe